targets/PythonSdk/source/playfab/testCompile.py.ejs

```diff
@@ -0,0 +1,16 @@
+import py_compile
+
+files = [
+<% for (var a = 0; a < apis.length; a++) { var api = apis[a];
+%>"PlayFab<%- api.name %>API.py",
```

Comment:
>api [](start = 14, length = 3)

Could just use apis[a] here without creating api each cycle?

Reply:
Yep, probably. I mostly write it this way because the for() { var ...; is written that way in every other template.